src/libs/rest.ts

```diff
@@ -0,0 +1,9 @@
+import express from "express";
+
+export const app = express();
+
+export function startServer() : void {
+  app.listen(3000, () => {
+    console.log("Listening on port 3000 new");
```

Comment:
You can use the logger that you have, instead of using console.log(). This is why you have the logger class in the first place.

Also what is 3000 new?

Reply:
done